Resources/Prototypes/Entities/Objects/Specific/chemistry-bottles.yml

```diff
@@ -87,34 +94,19 @@
   name: bottle
   parent: BaseChemistryEmptyBottle
   id: ChemistryEmptyBottle01
-  components:
-  - type: Sprite
-    sprite: Objects/Specific/Chemistry/bottle.rsi
-    layers:
-      - state: bottle-1
-      - state: bottle-1-1
-        map: ["enum.SolutionContainerLayers.Fill"]
-        visible: false
-  - type: Appearance
-  - type: SolutionContainerVisuals
-    maxFillLevels: 6
-    fillBaseName: bottle-1-
 
 - type: entity
   name: bottle
   parent: BaseChemistryEmptyBottle
   id: ChemistryEmptyBottle02
   components:
   - type: Sprite
-    sprite: Objects/Specific/Chemistry/bottle.rsi
     layers:
       - state: bottle-2
       - state: bottle-2-1
         map: ["enum.SolutionContainerLayers.Fill"]
         visible: false
-  - type: Appearance
   - type: SolutionContainerVisuals
-    maxFillLevels: 6
```

Comment:
Did you test if this worked? from a glance it looks like since you are overriding `SolutionContainerVisuals`, so `maxFillLevels` field would be overridden by its default which is 0, i could be wrong tho

Reply:
![snippy](https://user-images.githubusercontent.com/62370007/236602721-dbce7175-cd5d-434d-a578-25577bff14c8.PNG)
Seems fine, I figured that overriding a component will only effect the fields that you add in the child prototype, is this not the case? Removing the sprite path from the sprite component worked OK. 

I can re-add these if you want?

Reply:
yeah better be safe than sorry, directly declare maxFillsLevels when overriding SolutionContainerVisuals and it should be fine